### DISCUSSION TITLE
我买了两年的专业版，为啥我蓝灯上还是显示的免费版

### DISCUSSION THREAD

**Discussion by zhudamao123** [getlantern/lantern-forum-cn#753] [Q&A]
![蓝灯](https://github.com/user-attachments/assets/fa34d180-a3f6-44ce-bb58-bf741d79cc3e)
我前几天续了两年的专业版，但是支付宝充值后邮箱没有显示回馈，而且我的应用一直显示是免费版，怎么回事
![蓝灯](https://github.com/user-attachments/assets/3ebdebfb-6f39-4b71-9e28-93acab78d7b7)

**Comment by zhudamao123** [comment]
我的邮箱是982410136@qq.com

**Comment by MaxMKMax** [comment]
@zhudamao123 您好. 我已经帮您的问题提交上去了. 这两天请查看这个帖子或邮箱来获取激活码. 谢谢!

**Comment by zhudamao123** [comment]
邮箱发我吧



---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年8月6日(周二) 凌晨1:33
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 我买了两年的专业版，为啥我蓝灯上还是显示的免费版 (Discussion #753)




 
@zhudamao123 您好. 我已经帮您的问题提交上去了. 这两天请查看这个帖子或邮箱来获取激活码. 谢谢!

**Reply by zhudamao123** [reply]
发我邮箱吧，谢谢

**Reply by whpublic2024** [reply]
最后解决问题了么？

**Reply by zhudamao123** [reply]
什么时候可以发我激活码啊

**Comment by zhudamao123** [comment]
没有啊，激活码我还没收到啊



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年8月7日(周三) 上午10:15
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 我买了两年的专业版，为啥我蓝灯上还是显示的免费版 (Discussion #753)




 
最后解决问题了么？

**Reply by xiaolin6456** [reply]
你的电脑版可以正常使用吗，不闪退吗

**Reply by zhudamao123** [reply]
youtube会有，其余还好

**Reply by MaxMKMax** [reply]
982410136@qq.com
@zhudamao123 
您好. 已经给您以上的邮箱账号升级成功了. 问题应该已经解决. 谢谢您支持蓝灯!

**Reply by MaxMKMax** [reply]
@xiaolin6456 

您好. 请问您是有闪退的问题吗?

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by zhudamao123** [reply]
我用我邮箱登录，登不上去啊
![QQ截图20240808161329](https://github.com/user-attachments/assets/2d5b69f6-8af1-438a-adcb-059e30af30e5)

**Reply by MaxMKMax** [reply]
@zhudamao123 您好. 您是收不到验证码还是按钮登入不了?

**Comment by MaxMKMax** [comment]
@zhudamao123 您好. 您是收不到验证码还是按钮登入不了?

**Reply by zhudamao123** [reply]
哥们这是咋回事，激活码也不发我邮箱，验证码也不发我邮箱，我能申请退款吗

**Reply by zhudamao123** [reply]
不行你发我另一个邮箱吧，982410136qq@gmail.com吧

**Comment by zhudamao123** [comment]
收不到验证码啊



---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年8月8日(周四) 晚上11:56
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 我买了两年的专业版，为啥我蓝灯上还是显示的免费版 (Discussion #753)




 
@zhudamao123 您好. 您是收不到验证码还是按钮登入不了?

**Comment by zhudamao123** [comment]
验证码都申请了12个小时了，还是没有！



---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年8月8日(周四) 晚上11:56
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 我买了两年的专业版，为啥我蓝灯上还是显示的免费版 (Discussion #753)




 
@zhudamao123 您好. 您是收不到验证码还是按钮登入不了?

**Reply by zhudamao123** [reply]
我的邮箱还是没收到，咋回事？

**Comment by zhudamao123** [comment]
哥们，我信任你们，但你们也不能拿钱不办事吧，我们交了钱你总得给个解决方案吧，我现在根本登录不了

**Reply by zhudamao123** [reply]
蓝灯耍流氓
